src/main/java/org/generationcp/breeding/manager/listmanager/listcomponent/ListViewActionMenu.java

```diff
@@ -186,13 +186,13 @@ void setMessageSource(final SimpleResourceBundleMessageSource messageSource) {
 		this.messageSource = messageSource;
 	}
 
-	@PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_CROP_MANAGEMENT')")
+	@PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_GERMPLASM', 'ROLE_MANAGE_GERMPLASM', 'ROLE_SEARCH_GERMPLASM', 'ROLE_DELETE_GERMPLASM')")
```

Comment:
@cuenyad Should we split this function in 2? 
I am not sure why having ROLE_DELETE_GERMPLASM will enable Ungroup? 
I think both were together because in the past both were enabled for admins but we cant ensure that someone with ROLE_DELETE_GERMPLASM is an admin. 
I am trying to confirm with Mariano but I think that codingAndGroupingOptions should be enabled only for ROLE_ADMIN

Reply:
confirmed with Mariano. Please split this function in 2 @cuenyad